Submissions/engrshege.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "[Oluwasegun Idowu]",
+  "img": "[https://avatars.githubusercontent.com/u/52855925?v=4]",
+  "email": "[engrshege@gmail.com]",
+  "links": {
+    "website": "[https://engrshege.github.io/myportfolio/]",
+    "linkedin": "https://www.linkedin.com/in/engrshege",
+    "github": "https://github.com/engrshege"
+  },
+  "jobTitle": "[Frontend Engineer]",
+  "location": {
+    "city": "[Ikotun]",
+    "state": "[Lagos]",
+    "country": "[Nigeria]"
```

Comment:
Please also remove the brackets from here

Reply:
I have now made corrections by removing the brackets.

Thank you so much for your review ma'am.